doc/api/errors.md

```diff
@@ -1204,6 +1204,11 @@ The `inspector` module is not available for use.
 While using the `inspector` module, an attempt was made to use the inspector
 before it was connected.
 
+<a id="ERR_INSPECTOR_NOT_WORKER"></a>
+### ERR_INSPECTOR_NOT_WORKER
+
+An API was called on the main thread that can only be used from the Node worker
```

Comment:
An API was called on the main thread that can only be used from the worker thread.

Reply:
Done